config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/ContextVariableEnvironmentProperty.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.beans;
+
+import com.ca.apim.gateway.cagatewayconfig.config.spec.ConfigurationFile;
+import com.ca.apim.gateway.cagatewayconfig.config.spec.EnvironmentType;
+
+import javax.inject.Named;
+
+import static com.ca.apim.gateway.cagatewayconfig.config.spec.ConfigurationFile.FileType.PROPERTIES;
+
+@Named("CONTEXT_VARIABLE_ENVIRONMENT_PROPERTY")
+@ConfigurationFile(name = "env", type = PROPERTIES)
+@EnvironmentType("CONTEXT_VARIABLE_PROPERTY")
+public class ContextVariableEnvironmentProperty extends GenericEnvironmentProperty {
+
+    private String value;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "value" private field. ([squid:S1068](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=149&issues=AWep1xCAh8YERCev-dpP&open=AWep1xCAh8YERCev-dpP)

[ISSUE_KEY]: AWep1xCAh8YERCev-dpP